### DISCUSSION TITLE
Websockets / Nchan

### DISCUSSION THREAD

**Discussion by countrymusicfy** [AzuraCast/AzuraCast#4525] [Q&A]
If I enable Websockets in the System Setting, do I not need to install the nchan plugin?

--------------

Use WebSockets for Now Playing Updates
Enables or disables the use of the newer and faster WebSocket-based system for receiving live updates on public players. You may need to disable this if you encounter problems with it.

**Comment by Vaalyn** [comment]
You don't need to install anything for it to work. AzuraCast already includes nchan in it's nginx and will make use of it when enabled.

**Reply by abbadabbatech** [reply]
Do I need anything to use a websocket library for mobile?
